jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/timstorage/Circle.java

```diff
@@ -22,48 +22,39 @@
 
 @JsonPropertyOrder({ "center", "radius", "units" })
 public class Circle extends Asn1Object {
-   private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 1L;
 
-   private Position center;
+    @JsonProperty("center")
+    private Position center;
 
-   @JsonProperty("radius")
-   private String radius;
+    @JsonProperty("radius")
+    private String radius;
 
-   @JsonProperty("units")
-   private DistanceUnits units;
+    @JsonProperty("units")
+    private DistanceUnits units;
 
-   @JsonProperty("position")
```

Comment:
note: this is the offending annotation that caused the problem

Reply:
Can we add a unit test for this use case?

Reply:
Added!

Reply:
Is `position` an invalid property on a Circle Asn1Object in all cases, or is there any scenario where we want to have `position` present? I'm gathering that, at minimum, we don't want _both_ `position` and `center` present, but is there a scenario where we could want `position` present when `center` is not (and vice versa)?

Reply:
> Is `position` an invalid property on a Circle Asn1Object in all cases, or is there any scenario where we want to have `position` present? I'm gathering that, at minimum, we don't want _both_ `position` and `center` present, but is there a scenario where we could want `position` present when `center` is not (and vice versa)?

The ASN.1 representation for a Circle in J2735 only contains center, radius and units. I don't think we'll ever have a scenario where we want to have 'position' present.